mediagrains/numpy/numpy_grains/VideoGrain.py

```diff
@@ -222,6 +222,7 @@ class VideoGrain (bytesgrain.VideoGrain):
     _grain_conversions: Dict[Tuple[CogFrameFormat, CogFrameFormat], ConversionFunc] = {}
 
     def __init__(self,
+                 *args,
```

Comment:
NIT: Just want to double check this isn't an unintended breaking change. Do the other arguments still work as positional with this in place?

Reply:
They don't, but that's fine, as the 'old style' will call it from the `videograin.py` file which _does_ support positional arguments.